addons/mail/static/tests/helpers/mock_models.js

```diff
@@ -69,6 +69,7 @@ export class MockModels {
                 },
                 records: [
                     { icon: 'fa-envelope', id: 1, name: "Email" },
+                    { icon: 'fa-upload', id: 28, name: "Upload Document" },
```

Comment:
I don't like global records. This includes the activity type "Email" above, this is a mistake.
Each test should explicitly define the data. No magic number like `1` and `28`.

Reply:
I like global records. The goal of the mock server is to simulate the server, and the server has these records in data as a prerequisite for the system to work.

Reply:
What I don't like is global records that are used to lazily define a common setup to some tests, but here we are not in that case.

Reply:
Global records make sense when it's available by a single point of entry, not with arbitrary id and making use of this arbitrary id in the test `1`, `28`, etc.